### ISSUE TITLE
Accessibility fundamentals potential issues

### ISSUE THREAD

**Issue by mcoker** [patternfly/patternfly-org#3302]
Looking at https://www.patternfly.org/v4/accessibility/accessibility-fundamentals and I noticed some issues.

* "different user groups" link to https://www.a11yproject.com/posts/myth-accessibility-is-blind-people/ 404's
  * Looks like it changed to https://www.a11yproject.com/posts/accessibility-is-blind-people/
* > Users with poor motor control can use a range of devices to access contents. **
  * I don't see any sort of footnote for what the `**` may refer to
* > These needs mean that interfaces should not rely on color to communicate information
  * Maybe say "rely on color alone..."? We to use color as an indicator to communicate info, but in combination with some other visual indication
* > Often “the Persona Spectrum”
  * just looking around at how this is referred to, it looks like maybe this should be "Often the “Personal Spectrum“"
* > Don’t show interactive elements on hover. Interactive elements that can display in a pop-up must display on click/touch/Enter events.
  * I'm not sure if I have a specific recommendation, but the second part doesn't make much sense to me, but that could just be me!
* > There should be parity between hover and focus events.
  * This seems a little redundant with the first bullet point about parity between input types, but I might not see the difference.

The image:
1) There is no screen reader equivalent. The `alt` text just reads "Accessibility spectrum example". Assuming the image is meaningful, I think this goes against the experience parity principle. "There should be parity between the screen reader contents and visibly rendered contents"
2) The vertical text is hard to read
3) The text along the very bottom is so small, it's barely legible - looks like it's about 10px. I can't read it at all on my phone.
4) The links in the text at the bottom don't do anything since this is just a plain image
5) The native size is 6250x10629... and 5.3mb in size... there are all kinds of problems with that. At a minimum it should be resized to around 832px, which is the max size it will render currently. That said, if we ever widen the content area, the rendered size may change. If it hasn't already been, we should try and optimize its size. And we can deliver different images at different viewport sizes, and lazyload images, so they only load when they're brought into the viewport. I can help out here, just lemme know!
6) I wonder if it should come directly under the paragraph that talks about the personal spectrums represented in the image. Where it is now, it reads like it comes after the content for that section ends, under the last paragraph "There are plenty more examples we could include. Essentially, accessibility takes all users into account."

**Comment by mcoker** [comment]
Cool, thanks @jessiehuff! On the interactive element hover note - gotcha. I think I was confused about the interactive content part. Does that basically call out the difference between something like a tooltip versus popover - where the tooltip can be triggered on hover because it doesn't have interactive content, but the popover is triggered on click/touch/enter because (or when) it contains interactive content?

**Comment by jessiehuff** [comment]
Yup, exactly. I think that would definitely fit into this category. :)

**Comment by nicolethoen** [comment]
Closed by #3511
